internal/app/machined/pkg/controllers/secrets/kubernetes.go

```diff
@@ -26,6 +26,11 @@ import (
 	"github.com/talos-systems/talos/pkg/resources/v1alpha1"
 )
 
+// KubernetesCertificateValidityDuration is the validity duration for the certificates created with this controller.
+//
+// Controller automatically refreshes certs at 50% of CertificateValidityDuration.
+const KubernetesCertificateValidityDuration = constants.KubernetesDefaultCertificateValidityDuration
```

Comment:
I'm curious, why not just use the constant?

Reply:
I wanted to keep it local to this file as it's being used four times, to make sure it's in sync across all four usages. But no real reason besides that.